core-ui/src/components/Extensibility/components-form/NameRenderer.js

```diff
@@ -16,11 +16,13 @@ export function NameRenderer({
 }) {
   const { t: tExt } = useGetTranslation();
   const extraPaths = schema.get('extraPaths') || [];
+  const editMode = schema.get('editMode') || false;
 
   return (
     <K8sNameField
       value={value}
       kind={resource.kind}
+      readOnly={editMode}
```

Comment:
We don't need `editMode` in schema, if I use `widget: Name` in schema for input, it is disabling input in Create too. 

![image](https://user-images.githubusercontent.com/46759954/200277575-8114c5e5-ef34-4437-8843-94fe8846e4af.png)

We need that only in `ExtensibilityCreate` and `ResourceDetails`, like we had it in "old" implementation.